src/handlers/NumberGenerator.ts

```diff
@@ -38,4 +38,19 @@ export class NumberGenerator {
       return Promise.reject({statusCode: 500, body: error});
     }
   }
+
+  public async generateZNumber(): Promise<string> {
```

Comment:
I'm not sure if there are some unit tests for the trailerId which can be copied over